ldfparser/lin.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, frame_id: int, name: str, length: int, signals: Dict[int, Lin
 		self.publisher = None
 		self.length = length
 		orderedSignals = sorted(signals.items(), key=lambda x: x[0])
+		self.signal_map = orderedSignals
```

Comment:
This may need some clean up in the future, preferably it should be a single field.